api/USERS_GUIDE.md

```diff
@@ -281,4 +281,59 @@ current holds for the project:
     "P1.proj_waiting",
     "P1.proj_avail"
 ]
-```
\ No newline at end of file
+```
+
+### Proofreading
+
+The proofreading APIs require retaining and passing in state information. The
+following examples use [jq](https://jqlang.github.io/jq/) to parse responses
+and store them as environment variables.
+
+Get a project's details and parse out the project state:
+```bash
+curl -s -H "Accept: application/json" -H "X-API-KEY: $API_KEY" \
+    -X PUT "https://www.pgdp.org/api/v1/projects/projectID44de3936807f1" \
```

Comment:
This should be `GET`, shouldn't it? (Didn't notice it on my first readthrough, but sure noticed it just now trying to test it.)

Reply:
Yes it should, thanks for catching it.